python/README.md

```diff
@@ -37,6 +37,7 @@ $ poetry run pytest --cov=. tests
 The application current has three main functions:
 1. Ingest
 2. Ingest Status
+2. Ingest References
 3. Rewrite
 4. Chat
 5. Delete (References)
```

Comment:
It should say `seven main functions` now.
Also there are two number 2 in the list

Reply:
Good catch. Replaced to they auto-numbering with `1.` everywhere.